.gitignore

```diff
@@ -55,6 +55,16 @@ phpunit.xml
 
 # Logs
 /logs
+*.log
+npm-debug.log*
```

Comment:
I guess it wouldn't harm to ignore logs, but maybe data/csv files are needed sometimes under version control, not sure

Reply:
Good catch, Oleks. I've taken that list from the server. I will remove these and move them to a separate PR.

Reply:
Handled it in https://github.com/Automattic/woocommerce-payments/pull/6013/commits/3cf9c8b44db492d16c1e2ad8b0e65201b06744c9